specs/signing-and-verification-workflow.md

```diff
@@ -41,12 +41,12 @@ The user wants to pull an OCI artifact only if they are signed by a trusted publ
 
 ### Verification Steps
 
-1. **Should Notary v2 verify the signature? :** Depending upon [trust-policy](./trust-store-trust-policy.md#trust-policy) configuration, determine whether Notary v2 needs to verify the signature or not.
+1. **Should Notary Project specification implementor verify the signature? :** Depending upon [trust-policy](./trust-store-trust-policy.md#trust-policy) configuration, determine whether Notary Project specification implementor needs to verify the signature or not.
```

Comment:
Previously, `implementation` is used instead of `implementor`. We should be consistent on wording.